libraries/physics/src/ObjectAction.h

```diff
@@ -62,14 +64,10 @@ class ObjectAction : public btActionInterface, public EntityActionInterface, pub
     virtual void setAngularVelocity(glm::vec3 angularVelocity);
     virtual void activateBody();
 
-    bool _active;
     EntityItemWeakPointer _ownerEntity;
-
-    quint64 _expires; // in seconds since epoch
     QString _tag;
-
-    quint64 localTimeToServerTime(quint64 timeValue) const;
-    quint64 serverTimeToLocalTime(quint64 timeValue) const;
+    quint64 _expires { 0 }; // in seconds since epoch
+    bool _active { false };
```

Comment:
Cleanup of ObjectAction protected members and methods.  Nothing should have changed.